galaxy-simple/templates/galaxy_pvc.yaml

```diff
@@ -1,3 +1,4 @@
+{{ if .Values.galaxy_create_pvc }}
```

Comment:
You need to close this if at the end of what you want to enclose, with an `{{ end }}`